docs/docs/behaviors/index.mdx

```diff
@@ -0,0 +1,72 @@
+| `&sl`   | [Stickey Layer](sticky-layer.md)                        | A sticky layer stays pressed until another key is pressed                                               |
+
+## Mouse emulation behavior
+
+| Binding | Behavior                                                    | Description                    |
+| ------- | ----------------------------------------------------------- | ------------------------------ |
+| `&mkp`  | [Mouse Button Press](mouse-emulation.md#mouse-button-press) | Emulate pressing mouse buttons |
+
+## Reset behavior
+
+| Binding       | Behavior                          | Description                                                                              |
+| ------------- | --------------------------------- | ---------------------------------------------------------------------------------------- |
+| `&sys_reset`  | [Reset](reset.md#reset)           | Reset the keyboard and re-run the firmware flashed to the device                         |
+| `&bootloader` | [Bootloader](reset.md#bootloader) | Reset the keyboard and put it into bootloader mode, allowing you to flash a new firmware |
+
+## Output selection behavior
+
+| Binding | Behavior                                                 | Description                                                                                        |
+| ------- | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| `&bt`   | [Bluetooth](bluetooth.md#bluetooth-behavior)             | Completes a bluetooth action given on press, for example switching between devices                 |
+| `&out`  | [Output Selection](outputs.md#output-selection-behavior) | Allows selecting whether output is sent to the USB or bluetooth connection when both are connected |
+
+## Lighting behavior
+
+| Binding   | Behavior                                       | Description                                                                     |
+| --------- | ---------------------------------------------- | ------------------------------------------------------------------------------- |
+| `&rgb_ug` | [RGB Underglow](underglow.md#behavior-binding) | Controlling the RGB underglow, usually placed underneath the keyboard           |
+| `&bl`     | [Backlight](backlight.md#behavior-binding)     | Controlling the keyboard backlighting, usually placed through or under switches |
+
+## Power management behavior
+
+| Binding      | Behavior                                      | Description                                                     |
+| ------------ | --------------------------------------------- | --------------------------------------------------------------- |
+| `&ext_power` | [Power management](power.md#behavior-binding) | Allows enabling or disabling the VCC power output to save power |
+
+## Other behavior
```

Comment:
```suggestion
## User-defined behaviors
```
Does this make more sense? I am trying to make a distinction between pre-defined behavior bindings and custom/user-defined ones.